pagopa/pn-frontend
# PN-8692-fe-modificare-descrizione-evento-recrn-005-c-in-timeline

## Short description
Changed timeline events sentences for each languages of PA, PG and PF.

## List of changes proposed in this pull request
- changed sentence of `send-analog-flow-RECRN005C-description`
- changed sentence of `send-analog-flow-RECRN005C-description-multirecipient`

## How to test
Just yarn start pa/pf/pg and see if the sentence of both attributes have been changed.
